solr/core/src/test/org/apache/solr/filestore/TestDistribPackageStore.java

```diff
@@ -192,10 +192,10 @@ public static void checkAllNodesForFile(
       assertResponseValues(10, new Fetcher(url, jettySolrRunner), expected);
 
       if (verifyContent) {
-        try (HttpSolrClient solrClient = (HttpSolrClient) jettySolrRunner.newClient()) {
+        try (HttpSolrClient httpSolrClient = (HttpSolrClient) jettySolrRunner.newClient()) {
```

Comment:
solrClient was a totally fine name.  But rename if you wish.

Reply:
maybe the way to go is to stick to `solrClient` everywhere, instead of `solrServer`, `solrHttpClient`, etc... unless the name is somehow helps you understand what is going on...    Part of this refactor is listerally to move to less specific subclasses...

Reply:
Sure but don't bother doing renames for variables already declared to be of type SolrClient, as that would increase the scope massively.